presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManager.java

```diff
@@ -210,7 +125,6 @@ public void stop()
     {
         queryTracker.stop();
         queryManagementExecutor.shutdownNow();
-        unboundedExecutorService.shutdownNow();
```

Comment:
hmm? :)

Reply:
This is not just a rebase conflict, looks like all references to `unboundedExecutorService` (was: `queryExecutor`) should have been removed entirely.